content/ways-to-train.md

```diff
@@ -42,208 +39,89 @@ keywords:
+* [if you do not have a degree](#if-you-do-not-have-a-degree)
 
-You’ll be based in a school but you will not be employed by them.
+* [if you already have teaching experience](#if-you-already-have-teaching-experience)
 
-You’ll also be taught by the university, college, or school-centred initial teacher training (SCITT) provider that the school partners with.
+You can also find specialised programmes [if you're a career changer](#if-youre-a-career-changer) or [if you have a doctorate](#if-you-have-a-doctorate).
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+Not looking for primary or secondary teacher training? Consider teaching [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).
 
-You can also [get funding](/funding-your-training).
+### Return to university as a postgraduate student
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+If you already have a degree and want to go to university again, you can do a university-led PGCE or PGDE leading to qualified teacher status.
 
-$school-direct-salaried$
+A PGCE or PGDE is an academic qualification that you can use towards masters credits.
 
-You’ll be employed by a school as an unqualified teacher while you work towards your Qualified Teacher Status (QTS).
+You spend about a third of your time in academic learning and the rest in school placements.  
 
-You’ll earn a salary and do not need to pay any tuition fees.
+Courses typically take one year but you may be able to study part time.
 
-This route typically requires 3 years of transferable work history.
+Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects.
 
-Most school direct programmes lead to a PGCE, but not all do. Check with the training provider before you apply.
+You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-loans).
 
-<p class="call-to-action__action">
-  <a href="https://www.gov.uk/find-postgraduate-teacher-training-courses">Find a school direct <span>course</span></a>
-</p>
+<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find university training</a>
 
-$postgraduate-teaching-apprenticeship$
+### Learn on the job in schools near you
 
-You’ll be employed by a school and learn while you’re working.
+If you already have a degree and want to spend as much time as possible in practical learning, you can find a training programme led by a school or group of schools.
 
-You’ll split your time between school and university study, with at least 20% of your time studying.
+You'll only spend about about a fifth of your time in academic learning (for example, written assignments).
```

Comment:
I would assume academic learning being in-class (whether virtual or not) time, and written assignments being additional to that in one's own time?